client/src/app/services/websocket-api/websocket-api.service.ts

```diff
@@ -58,7 +79,20 @@ export class WebsocketAPIService {
   }
 
   onMessageReceived(message): void {
-    console.log('Message Recieved from Server :: ', message);
-    // will dispatch actions to the storeFacade when recieving messages
+    this.count += 1;
+    console.log('Message Recieved from Server :: ', JSON.parse(message.body));
+    if (this.userId && message && message.body && message.body) {
+      const snackBarRef = this.snackBar.open(
+        'The status of one of your orders has been updated',
+        'View',
+        { duration: 5000 }
+      );
+      snackBarRef.onAction().subscribe(() => {
+        // currently the profile page does not exist but when it does, it will guide the users there
+        // this.router.navigate(['profile'])
+      });
+      this.storeFacade.getCurrentUser(this.userId);
+    }
+    console.log('COUNT DEBUG: ', this.count);
```

Comment:
Do we want to change these console.logs in future commits?

Reply:
Yeah this should be changed.

Reply:
Update: in future commits, we can change that. It does not impact SonarCloud or any implementation. Since we`re gonna deploy that in pre-prod (uat/dev), there is nothing to worry about for this console.log printing.